browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -54,6 +54,22 @@ using mojom::CharacterType;
 using mojom::ConversationTurn;
 using mojom::ConversationTurnVisibility;
 
+AIChatUIPageHandler::ChatContextObserver::ChatContextObserver(
+    content::WebContents* web_contents,
+    AIChatUIPageHandler* page_handler)
+    : content::WebContentsObserver(web_contents), page_handler_(page_handler) {}
+
+AIChatUIPageHandler::ChatContextObserver::~ChatContextObserver() = default;
+
+void AIChatUIPageHandler::ChatContextObserver::WebContentsDestroyed() {
+  page_handler_->HandleWebContentsDestroyed();
+}
+
+void AIChatUIPageHandler::HandleWebContentsDestroyed() {
```

Comment:
pls put this method lower in `.cc` file (the order should be roughly the same as it's declared in `.h`).

Reply:
fixed.